tests/admin/import/test-class-import-detector.php

```diff
@@ -11,6 +11,8 @@
 class WPSEO_Import_External_Detector_Test extends WPSEO_UnitTestCase {
 
 	/**
+	 *  Test whether we can return proplery when do not detect a plugin's data.
```

Comment:
Spelling: `properly`
Also, this sentence seems to have taken a wrong turn somewhere.